### ISSUE TITLE
Cracking Speed never uptaded and is 0.0 H/s

### ISSUE THREAD

**Issue by zip62** [hashview/hashview#448]
Hello,

I use Ubuntu 16.04 and the last github : https://github.com/hashview/hashview.git

I have a problem, "cracking speed" is never updated and value is all time 0.0 H/s :

![capture](https://user-images.githubusercontent.com/6962161/46255171-0bfdf300-c49a-11e8-8d9c-33b1febba79a.PNG)

In the console view, i have : "10:12:02 hashcat-worker.1    | Speed.#1.........:  2136.1 MH/s (11.40ms) @ Accel:128 Loops:32 Thr:1024 Vec:1"

![capture](https://user-images.githubusercontent.com/6962161/46255137-bcb7c280-c499-11e8-8dea-0787a84cc005.PNG)

Do you know how i can fix that ?

Thanks,
Mathieu

**Comment by zip62** [comment]
Hum...

New installation, same problem. And found hash not updated too (see yellow in picture).

![capture](https://user-images.githubusercontent.com/6962161/47257562-c5dff200-d48f-11e8-8baa-36ce338f50e7.PNG)

It's a bug or i have a problem with my installation ?

Thanks,
Regards.

**Comment by HugoPouliquen** [comment]
I tried again from scratch and it's working! ![image](https://user-images.githubusercontent.com/7288882/47300390-9c080600-d61c-11e8-88b9-0e2d6963c9c1.png)

What is your ruby version? Remove and try `v0.7.4-beta` branch with ruby 2.4.4

**Comment by zip62** [comment]
Hello,

I used "ruby-2.2.2" with v0.7.3.

Thanks,
Regards.

**Comment by HugoPouliquen** [comment]
Hello :)

Ok remove and test with v0.7.4, and tell me if you still have the same problem.

Regards.

**Comment by zip62** [comment]
Hello,

I have the same problem, cracking speed is 0.0 H/s.

Thanks,
Regards.
![capture](https://user-images.githubusercontent.com/6962161/47307175-aaf7b400-d62e-11e8-840d-4558ab4ed396.PNG)

**Comment by i128** [comment]
I've run into similar issues with this. Are you using a single or multiple agents?
with hashview stopped, is hashcat still running (i.e. via ps)

**Comment by zip62** [comment]
I'am using a single agent.
I have uninstalled this tool, i can't test anymore. Sorry.

**Comment by BenKettlewell** [comment]
I'm looking into this issue as well. Single local agent but 4 GPUs, is it not able to sum them together?

I found [this stdout parser](https://github.com/hashview/hashview/blob/1d4dce6f3d9cbb7056f829f84b7c6283a878ea6e/helpers/hc_stdout_parser.rb#L17) which is looking for Speed.Dev and then is later ingested [here](https://github.com/hashview/hashview/blob/2f9e3dd97ccd6e6f1ea2fb129ef322bf2b122883/routes/api.rb#L310).

`
if item.to_s =~ /Speed Dev #/
`

I think it's looking for the lines below but I'm using hashcat v5.1. Looks like it should be Speed.#* and it's already converted to human readable format in MH/s so I'm not sure what the conversion lines are for.

![image](https://user-images.githubusercontent.com/7477112/50370938-84d68d80-0565-11e9-973e-ca1030337d1c.png)

**Comment by i128** [comment]
Yup! I just ran into this recently too. I have some code to fix it. It's
just not committed yet. Hopefully will get to it soon.
